e2e/__tests__/__snapshots__/detectOpenHandles.ts.snap

```diff
@@ -13,7 +13,7 @@ This usually means that there are asynchronous operations that weren't stopped i
 exports[`prints out info about open handlers 1`] = `
 Jest has detected the following 1 open handle potentially keeping Jest from exiting:
 
-  ●  DNSCHANNEL
+  ●  TCPSERVERWRAP
```

Comment:
This is actually a perfect demonstration of the issue where meaningless `DNSCHANNEL` warnings could obscure the other indirectly created handles that are actually causing problems. 😄 

I’m wondering if it might be a good follow-on PR to alter the stack traces for indirectly created async resources or the deduplication code slightly so that we log all the handles indirectly created from the same spot in user code, rather than just one.

Stack traces for indirectly created handles:
https://github.com/facebook/jest/blob/ae3fc943d48a8b690e7cd5872f652c6aaef91ddb/packages/jest-core/src/collectHandles.ts#L80-L86

Deduplicating based on stack trace:
https://github.com/facebook/jest/blob/ae3fc943d48a8b690e7cd5872f652c6aaef91ddb/packages/jest-core/src/collectHandles.ts#L151-L171

Reply:
It looks somewhat noisy with duplicated traces. But I guess we might miss out on the more "descriptive" handle? I'm open to a PR changing this with a good example of where it improves the output 🙂

Reply:
> It looks somewhat noisy with duplicated traces.

For sure. Will play around with this and see how it looks or maybe tweak the formatting over the weekend. 👍